tsdb/engine/tsm1/file_store.go

```diff
@@ -207,7 +211,7 @@ func (f *FileStore) Open() error {
 		return nil
 	}
 
-	files, err := filepath.Glob(filepath.Join(f.dir, fmt.Sprintf("*.%s", "tsm1dev")))
+	files, err := filepath.Glob(filepath.Join(f.dir, fmt.Sprintf("*.%s", "tsm")))
```

Comment:
Perhaps this extension should be a const somwhere. So we prevent stupid bugs like tsm vs tsm1 for the file extension.